theme/style.css

```diff
@@ -9,10 +9,50 @@
   background-position: center;
 }
 
+
+
 .header h1 {
   display: block;
   max-width: 750px;
   text-align: center;
 }
 
+img {
```

Comment:
these selectors are too broad. They will apply to every image on the page, which wouldn't be correct.